.circleci/config.yml

```diff
@@ -265,6 +273,11 @@ jobs:
       - run_karma:
           disable_synthetic: true
           disable_aria_reflection_polyfill: true
+      - run_karma:
+          node_env_production: true
+      - run_karma:
+          node_env_production: true
+          disable_synthetic: true
```

Comment:
We could do the combinatorial explosion here, but I figure synthetic + native gives us pretty good test coverage.